ClinicWeb/src/main/java/pl/infoshare/clinicweb/user/registration/RegistrationController.java

```diff
@@ -46,6 +46,35 @@ public String registerFormSubmission(@Valid @ModelAttribute("user") UserDto user
         return "redirect:/login";
     }
 
+    @GetMapping("/register-admin")
+    public String registerByAdmin(@ModelAttribute UserDto user, Model model) {
+
+
+        model.addAttribute("user", user);
+
+        log.info("New user registration form was requested.");
+
+        return "user/registry-admin";
+    }
+
+    @PostMapping("/register-admin")
+    public String userCreatedByAdmin(@Valid @ModelAttribute("user") UserDto user, BindingResult bindingResult,
+                                     Model model, RedirectAttributes redirectAttributes) {
+
+        if (bindingResult.hasErrors()) {
+
+            model.addAttribute("user", user);
+            log.info("Validation error occured while registering user.");
+            return "user/registry-admin";
+        }
+
+        userService.saveUser(user);
+        log.info("User was successfully registered by Admin {}", user.getRole(), user.getEmail());
```

Comment:
```suggestion
        log.info("Successfully registered user with given eamil: {}",, user.getEmail());
```
Produkcyjnie dodawanie zbędnych informacji powoduje przepełnienie logu, najlepiej podawać inforamcje nieżbedne do identyfikacji jak np Id lub email kiedy jest unikatowy. 
Dodatkowo kiedy logujesz informację i używasz "{}" , a nastepnie nadajesz parametry np. user.getRole() musisz mieć tyle samo nawiasów co parametrów. Powyżej masz jeden nawias i dwa parametry w tym przypadki user.getEmail nie zostanie zalogowane.